src/validators/comment-validator.js

```diff
@@ -0,0 +1,27 @@
+const  {check}  =  require ( 'express-validator' );
+const {validationAccountId, validationComment} = require('../validators/discussion-validator');
+const {validationId} = require('../validators/account-validator');
+const {validateResult} = require('../helpers/response-result');
+
+const validationCreationComment = [
+    validationComment,
+    validationAccountId,
+    check('idDiscussion')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .matches(/^[a-z0-9]{24}$/).withMessage('El ID debe tener números y letras minúsculas')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
        .matches(/^[a-z0-9]{24}$/).withMessage("El ID debe tener números y letras minúsculas")
```